### ISSUE TITLE
Wonky highlighting after '' *sometimes*

### ISSUE THREAD

**Issue by jeaye** [LnL7/vim-nix#7]
I've been able to reproduce this using this text (the preceding newlines are intentional): 
https://gist.github.com/jeaye/3b75f72ecd266e6fde6f

It may not happen at first, but scrolling (with the mouse seems to help) down to the bottom, then scrolling back to the top (not with gg). Once you're at the top, click the first line to reset the cursor (again, not gg) and scroll back down again.

These steps are crazily convoluted and the problem must be simpler, but this is what I have right now; let me know if you can reproduce it.

**Comment by rnhmjoj** [comment]
Yeah, I can confirm. With the method you described I can reproduce it but I rarely scroll with the mouse and happens to me a lot anyway, so it must be triggered by something more general.

**Comment by rnhmjoj** [comment]
Also happens when there is a string like this:
```
buildPhase = ''
  [...]
  patchelf --set-rpath "$libPath" binary
'';
```
after I remove the quotes from `$libPath`

**Comment by aszlig** [comment]
Hm, I can't really reproduce this, what exactly is "wonky" there? Do you have a screen shot, or even better would be a Vader test that actually reproduces the issue.

**Comment by nicknovitski** [comment]
I had this problem (and others), but it turned out to be [a conflict with the default settings of the `vim_configurable` package](http://nicknovitski.com/vim-nix-syntax).  If you're installing Vim with plugins via Nix, I'd recommend disabling the built-in nix filetype "plugin" in your `config.nix`.

**Comment by rnhmjoj** [comment]
You exactly described my problem but I'm not using plugins from nix, vim_configurable or vim (neovim).

**Comment by nicknovitski** [comment]
Darn.  The underlying cause for me was that a nix filetype was being secretly defined in the runtime folder, but it seems unlikely you've got the same problem.  Sorry I couldn't help.
